src/service/owners-impl.js

```diff
@@ -108,23 +108,9 @@ export class OwnersImpl {
         return;
       }
       if (resource.getState() != ResourceState.LAYOUT_SCHEDULED) {
-        promises.push(
```

Comment:
This refactor could be done separately of the rest of this PR right? Its a shift from `whenBuild().then...` to `.build().then(() => layout())`